roles-in-tech/security.md

```diff
@@ -1,94 +1,164 @@
 # Security
 
+![security locks](\roles-in-tech\security-locks.jpg)
+
 ### Projected Time
 90-180 minutes
 
 ### Prerequisites
-How the Internet Works Topic Outline
+- To become security engineer, it is recommended to  pursue bachelors degree in computer or have experience in security related field. Because employers prefer those candidates.
+
+- [How the Internet Works Topic Outline](https://github.com/Techtonica/curriculum/blob/master/requests-and-responses/requests-and-responses.md)
+
+- [Intro to Security](https://github.com/Techtonica/curriculum/blob/master/security/web-security.md)
 
 ### Motivation
+- Computer security is important, primarily to *keep your information protected*. It’s also important for your computer’s overall health, helping to prevent viruses and malware and allowing programs to run more smoothly.
```

Comment:
*malwares

Reply:
I have changed the file but I am facing problem in pushing changes in the merged branch. How can I push committed message?

Reply:
It is already merged. But don’t worry about it; I think malware is already plural, and is correct.

Reply:
Okay.
I though if it's written viruses then it should be malwares too.

Reply:
If you have 2 instances of software, do you now have softwares?

Reply:
Okay, I got it and by the way virus is also a kind of software.